Tests/scripts/run_private_build_tests.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
how is it different from the one we have in content ?

Reply:
you can merge this with the current script.
extract the relevant changes as script arguments (CircleCI vs. GitHub actions)